### DISCUSSION TITLE
How do I only hide the title bar without compromising on the round corners and the "headlights" in MacOS?

### DISCUSSION THREAD

**Discussion by rosnerdev** [tauri-apps/tauri#4088] [Q&A] [answered]
The only way to hide the title bar I found is to do this:
```javascript
"windows": [
      {
        "decorations": false
      }
```
But instead of only hiding the title bar, it cancels the round corners and the "headlights"...
Does anyone know what I should do?

**Comment by JonasKruckenberg** [comment, marked as answer]
You can copy paste this solution here: https://github.com/tauri-apps/tauri/issues/2663#issuecomment-1088906114 until PR #3965 is merged.

**Reply by tiagozip** [reply]
Unfortunately that doesn't work for windows.

**Reply by FabianLars** [reply]
windows doesn't have similar OS apis. For the window controls these are basically always custom drawn in apps that don't use the native titlebar. In the context of tauri apps you'd use simple html/css (the buttons are part of the windows system fonts, can't remember the name rn).
For the native borders (and rounded corners on win11) you can use https://github.com/tauri-apps/window-shadows/)
